tests/unit/test-api.c

```diff
@@ -923,5 +923,61 @@ main (void)
     jerry_cleanup ();
   }
 
+  /* Save literals */
+  if (jerry_is_feature_enabled (JERRY_FEATURE_SNAPSHOT_SAVE))
+  {
+    /* C format generation */
+    jerry_init (JERRY_INIT_EMPTY);
+
+    static jerry_char_t literal_buffer_c[256];
+    static const char *code_for_c_format_p = "var object = { aa:'min', Bb:'max', aaa:'xzy0' };";
+
+    size_t literal_sizes_c_format = jerry_parse_and_save_literals ((jerry_char_t *) code_for_c_format_p,
+                                                                   strlen (code_for_c_format_p),
+                                                                   false,
+                                                                   literal_buffer_c,
+                                                                   sizeof (literal_buffer_c),
+                                                                   true);
+    TEST_ASSERT (literal_sizes_c_format == 203);
```

Comment:
That is the size of the C-style format, please look 3 lines below for the expected result.